pykokkos/lib/ufuncs.py

```diff
@@ -0,0 +1,61 @@
+import pykokkos as pk
+
+# NOTE: it might be slick if we could encapsulate
+# the workload code inside of the ufunc definitions,
+# but we currently require global scoping
+
+
+@pk.workload
```

Comment:
I feel like we should just use a standalone workunit here instead of a workload. It reduces overhead slightly since we won't be creating an instance of the workload